examples/dev/nephos_config.yaml

```diff
@@ -2,7 +2,7 @@ core:
   # Comment out to make it work on your cluster
   cluster: minikube
   # ChartMuseum repository or directory holding charts
-  chart_repo: stable
+  chart_repo: aidtech
```

Comment:
We should set this to "stable", as others won't have access to `aidtech` chart repo.

Reply:
Thanks